config/environments/development.rb

```diff
@@ -48,16 +48,16 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,config/environments/development.rb
+  # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   #EMAIL
   #EMAIL_FROM = 'email_from@example.com'
   #EMAIL_TO = ['email_to1@example.com', 'email_to2@example.com']
 
-  #Password for request
-  PASSWD = 'changelater'
+  #Password for sending request from curl command
+  PASSWD = ENV['password']
```

Comment:
This (and the one in production.rb) could be removed and instead, the one place where you read PASSWD (jobs_controller.rb line 35) could read if from the array there.